scripts/pkgdep.sh

```diff
@@ -34,11 +36,12 @@ INSTALL_CRYPTO=false
 INSTALL_DEV_TOOLS=false
 INSTALL_PMEM=false
 INSTALL_FUSE=false
+INSTALL_RBD=false
 INSTALL_RDMA=false
 INSTALL_DOCS=false
 INSTALL_LIBURING=false
 
-while getopts 'abdfhipru-:' optchar; do
+while getopts 'abdfhipruDR-:' optchar; do
```

Comment:
```suggestion
while getopts 'abdfhipruR-:' optchar; do
```
If you're wondering how I spotted this one... is 'cause I didn't realize that you already submitted this PR and also cherry-picked the RBD changes on my own 🤦🏼‍♂️

Reply:
got it. I took the commit just to make sure we can map the commit from spdk upstream to this branch of spdk.